aunt-leahs-app/src/App.js

```diff
@@ -2,11 +2,25 @@ import React from 'react';
 
 import './App.css';
 import HomePage from './pages/homepage/homepage';
+import TextInput from './components/textField/textField.jsx';
+import './styles.css'
+
+const handleChange = (e) =>{
+  console.log('handleChange Called')
+  console.log(e.target.value)
+};
```

Comment:
handleChange should probably call setState, right?

Reply:
Will remove that part. Was meant for testing.